sources/packages/api/src/database/entities/education-program-offering.model.ts

```diff
@@ -14,7 +14,11 @@ import { OfferingTypes, User, OfferingStatus, Note } from ".";
 import { OfferingIntensity } from "./offering-intensity.type";
 import { dateOnlyTransformer } from "../transformers/date-only.transformer";
 
-export const OFFERING_NAME_MAX_LENGTH = 300;
+/**
+ * Max value the offering name can have. By DB definition it is defined as
+ * 300 but from business perspective it should not be longer than 50.
+ */
```

Comment:
Nothing major, but any particular reason why the DB definition and business requirement of 50 should not be the same ?

Reply:
I can chime in, it was originally 300 then changed in the UI to be 50 because it's more reasonable, it's not a hard requirement so we didn't feel the need to change it to match and it's possible it may need to be changed. After testing if it should be longer it's a simple fix

Reply:
Thanks for the clarification @abschwenker .